src/Oracles/Config/Setting.hs

```diff
@@ -133,6 +148,25 @@ osxHost = anyHostOs ["darwin"]
 windowsHost :: Action Bool
 windowsHost = anyHostOs ["mingw32", "cygwin32"]
 
+-- https://github.com/ghc/ghc/blob/master/mk/config.mk.in#L241
+relocatableBuild :: Action Bool
+relocatableBuild = windowsHost
+
+-- TODO: Prettify this into a builder?
+installDir :: String -> Action ()
```

Comment:
I mean Haddock documentation to clarify the purpose of each action. Here is an example:
```haskell
-- | Copy a file tracking the source, create the target directory if missing.
copyFile :: FilePath -> FilePath -> Action ()
copyFile source target = do
...
```